python/installer/__main__.py

```
@@ -0,0 +1,306 @@
+    disks: Sequence[str],
+    swap_size: int,
+    reserve: int,
+    encrypt_key: str | None,
+) -> None:
+    """Main."""
+    logging.info("Starting installation")
+
+    for disk in disks:
+        partition_disk(disk, swap_size, reserve)
+
+        test = Popen(("printf", f"'{encrypt_key}'"), stdout=PIPE)
```

Ruff (0.14.2)

`subprocess` call: check for execution of untrusted input

(S603)